models/const.go

```diff
@@ -55,4 +55,7 @@ const (
 	StockTypePaperLetter = "PAPER_LETTER"
 	StockTypePaper4x6    = "PAPER_4X6"
 	WeightUnitsLB        = "LB"
+
+	MaximumWeightInLbs       = 150.0
+	SafeGuardForZeroWeightOz = 15
```

Comment:
are we changing the safeguard weight from 13 to 15?

Reply:
**3. if no default weights available or if item count is unavailable, 13oz should be used for the entire return package in rate call**